hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx

```diff
@@ -71,14 +72,9 @@ class NavBar extends React.Component<INavBarProps> {
     });
   };
 
-  refresh = () => {
-    console.log("refresh");
-    this.props.history.push('/Heatmap');
-    window.location.reload();
-    }
-
   render() {
     const {location} = this.props;
+    const { isHeatmapAvailable } = this.state;
```

Comment:
@smitajoshi12 thanks for working on this patch. However can we keep the variable naming consistent. Here we have defined as "`isHeatmapAvailable`" and in heatmap.tsx file, we have defined as `isHeatmapEnabled`

Reply:
@devmadhuu 
updated variable name as per suggestion.